README.md

```diff
@@ -17,3 +17,11 @@ $ mope <command> [<args>]
 - [build](docs/build.md)
 - [run](docs/run.md)
 - [tx](docs/tx.md)
+
+
+## Developing
+When developing run the command you are working on in a repo that has a project defined (e.g. `mup-web`). 
```

Comment:
Sorry to nitpick wording, but maybe start at a higher level:

"The `mwp-cli` executable (mope) is defined in `/bin/mope.js` - you can execute it directly without installing the `mwp-cli` package. In development, this means that you can test the CLI from your application directory by running `node path/to/mwp-cli/bin/mope.js`

Example:

1.
2.
3.